### PR TITLE
Add release notes for Dashboards 1.3.0

### PR DESCRIPTION
**Pull request by kavilla**
Signed-off-by: Kawika Avilla <kavilla414@gmail.com>

### CODE REVIEW

**Review by ananzh — APPROVED** [review]

**Comment by tmarkley** [comment]
Should we merge this to `main` first, then `1.x` and `1.3`?

**Comment by tmarkley** [comment]
> > Should we merge this to `main` first, then `1.x` and `1.3`?
> 
> 
> 
> Would you be opposed to starting in 1.x then backporting that to 1.3 and then forwarding that to main? The process might a be very manual since from what I can tell git-release-notes does just a range and I will have to remove the ones from 2.0.

I guess if we have commits that only went to 1.x and not main then that makes sense.

File: release-notes/opensearch-dashboards.release-notes-1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+    
+    * [Build] Build ARM64 for deb and rpm
+     Build ARM64 for deb with the following commands:
+    yarn build --deb-arm
+    --skip-archives
+    yarn build --deb-arm --skip-archives --release
+     Build ARM64 for rpm with the following commands:
+    yarn build --rpm-arm
+    --skip-archives
+    yarn build --rpm-arm --skip-archives --release
```
Comment (AMoo-Miki) [inline release-notes/opensearch-dashboards.release-notes-1.3.0.md:175]: Would look nicer if wrapped in ```code```

**Review by AMoo-Miki — APPROVED** [review]

**Comment by kavilla** [comment]
Replacement: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1348

**Pull request closed without merging**